lib/PlacesProvider.js

```diff
@@ -194,17 +194,25 @@ Links.prototype = {
       limit = HISTORY_RESULTS_LIMIT;
     }
 
-    let sqlQuery =  `SELECT moz_places.url as url, moz_favicons.data as favicon, mime_type as mimeType,
-                            title, frecency, last_visit_date as lastVisitDate,  "history"  as type
+    let sqlQuery =  `SELECT moz_places.url as url,
+                            moz_favicons.data as favicon,
+                            mime_type as mimeType,
+                            moz_places.title,
+                            frecency,
+                            last_visit_date as lastVisitDate,
+                            "history"  as type,
+                            moz_bookmarks.id as bookmark
```

Comment:
@mzhilyaev here, i looked it up:

https://dxr.mozilla.org/mozilla-central/rev/5b2baa5e9356644a7ed0b73e422eaff62e159ffb/toolkit/components/places/Bookmarks.jsm#381

We need the bookmark GUID instead of theID.